go.mod

```
@@ -27,6 +28,7 @@ require (
 	k8s.io/component-helpers v0.29.2
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/controller-runtime v0.17.2
+	sigs.k8s.io/knftables v0.0.15
```

```suggestion
	sigs.k8s.io/knftables v0.0.16
```

(released after you submitted this).